<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Heap Visualizer</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>381</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="serverSettingsBtn">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>290</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sahadeva</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sahadeva</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="localBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="serverBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
